developer/bake_config.py

```diff
@@ -177,11 +156,6 @@ def bake_config(
                 bundle_cache_root_override=bundle_cache_root,
             )
             associated_core_desc.ensure_local()
-        else:
```

Comment:
We're not reporting things we do not cache when caching apps, engines and frameworks. we shouldn't be for the core either.